oca_dependencies.txt

```diff
@@ -27,6 +27,6 @@ server-ux
 # geospatial
 # report-print-send
 # server-tools
-# website-cms
-# website
+website-cms
+website
```

Comment:
Use our fork which contains the website_no_index module
```suggestion
website 'https://github.com/CompassionCH/website' 11.0
```
There are conflicts with the devel branch on this file.